cli/src/main/scala/edu/scalanus/cli/Repl.scala

```diff
@@ -0,0 +1,64 @@
+package edu.scalanus.cli
+
+import javax.script.ScriptEngineManager
+
+import org.jline.reader.{EndOfFileException, LineReaderBuilder, UserInterruptException}
+import org.jline.terminal.{Terminal, TerminalBuilder}
+
+class Repl {
+
+  private val engine = new ScriptEngineManager().getEngineByName("scalanus")
+
+  def main(config: ArgConfig): Unit = {
+    println("Welcome to " + detailedVersion(engine))
+    println("Press ^D or ^C twice to exit.")
+
+    val terminal = TerminalBuilder.builder().name("Scalanus")
```

Comment:
To pytanie czemu ten drugi napis jest z małej? w sensie po co robić raz tak, raz tak?